### PR TITLE
Add awesome stuffs

### PR DESCRIPTION
**Pull request by natelistrom**

### CODE REVIEW

**Comment by natelistrom** [comment]
@natelistrom does this recognize @mentions?

**Comment by natelistrom** [comment]
@natelistrom you need to fix #4

**Comment by natelistrom** [comment]
Closing this as it is not helpful.

**Comment by natelistrom** [comment]
# MOAAAARRR text!

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ A longer description
 Extra stuff.
 
 (by one character)
```
Comment (natelistrom) [inline README.md:8]: Whitespace

File: README.md [hunk 2]
```diff
@@ -6,3 +6,5 @@ A longer description
...
+
+Here's some more stuffs.
```
Comment (natelistrom) [inline README.md:10]: MOAR whitespace

**Review by natelistrom — COMMENTED** [review]
whitespace

**Review by natelistrom — COMMENTED** [review]
Comment

File: README.md (hunk 1 above)
Comment (natelistrom) [reply README.md:8]: This is just silly.

File: README.md (hunk 2 above)
Comment (natelistrom) [reply README.md:10]: This is serious

File: README.md [hunk 3]
```diff
@@ -6,3 +6,5 @@ A longer description
 Extra stuff.
 
 (by one character)
+
```
Comment (natelistrom) [inline README.md:9]: Starting a review...

File: README.md (hunk 2 above)
Comment (natelistrom) [inline README.md:10]: Adding a review comment...

File: README.md [hunk 4]
```diff
@@ -6,3 +6,5 @@ A longer description
 Extra stuff.
```
Comment (natelistrom) [inline README.md:6]: Adding a second review comment.

**Review by natelistrom — COMMENTED** [review]
You should add more foo to the bar.

File: README.md (hunk 4 above)
Comment (mmitrik) [reply README.md:6]: I agree with you

File: README.md (hunk 1 above)
Comment (mmitrik) [reply README.md:8]: +1 (start review)

File: README.md (hunk 3 above)
Comment (mmitrik) [reply README.md:9]: ok

File: README.md (hunk 2 above)
Comment (mmitrik) [reply README.md:10]: not important

Comment (mmitrik) [inline README.md:10]: I also notcied this thing Nate didn't comment on

{edits lorem ipsujm }

**Review by mmitrik — CHANGES_REQUESTED** [review]
Here's my feedback.  take it or leave it

**Comment by mmitrik** [comment]
put my feedback here

**Pull request closed without merging**